nft/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "pallet-nft"
-version = "6.0.2"
+version = "6.1.0"
```

```suggestion
version = "7.0.0"
```
This is breaking change